app/views/users/new.html.erb

```diff
@@ -0,0 +1,50 @@
+=begin 
```

Comment:
It is not being commented out because Html does not understand `=begin` and `=end`. IT is a ruby thing 😛 . 

Please address 😅

Reply:
You could have also simply used HTML comment syntax. Like:
```html
<!-- <form accept-charset="UTF-8" action = "/users" method="POST" >
    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
    <label for="user_name">Username</label>
    <input id="user_name" name="user[username]" type="text" /> <br>
    <label for="user_email">Email</label>
    <input id="user_email" name="user[email]" type="email" /> <br>
    <label for="user_password">Password</label>
    <input id="user_password" name="user[password]" type="password" /> <br> 
    <input type="submit" name ="submit" value="Submit">
</form> -->
```